File: easy/26/26.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        l = 1
        if len(nums) == 0:
            return 0
        for r in range(len(nums) - 1):
            if nums[r] != nums[r + 1]:
                nums[l] = nums[r + 1]
                l += 1
        return l
# Time Complexity O(n)
# Space Complexity O(1)
            

File: easy/26/26.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int l=1,r=0,length=nums.length-1;
        while (r<length){
            if (nums[r] != nums[r + 1]){
                nums[l] = nums[r + 1];
                l++;
            }
            r++;
        }
        return l;
    }
        
}
// Time complexity O(n)
// Space complexity O(1)
